src/components/views/rooms/EventTile.js

```diff
@@ -621,6 +628,7 @@ module.exports = withMatrixClient(React.createClass({
                                 { timestamp }
                             </a>
                             { this._renderE2EPadlock() }
+                            { Reply.getQuote(this.props.mxEvent, this.props.onWidgetLoad) }
```

Comment:
This should fallback to `<div />`

Reply:
https://reactjs.org/docs/conditional-rendering.html#preventing-component-from-rendering seems to suggest otherwise

Reply:
OK, fair enough. I wonder where our general paranoia against using `null` has come from..